draft-ietf-mls-architecture.md

```diff
@@ -894,8 +900,14 @@ Additionally, some services require that a recipient be able to prove to the
 service provider that a message was sent by a given client, in order to report
 abuse. MLS supports both of these use cases. In some deployments, these services
 are provided by mechanisms which allow the receiver to prove a message's origin
-to a third party (this if often called "non-repudiation"), but it should also be
-possible to operate MLS in a "deniable" mode where such proof is not possible.
+to a third party. This is often called "non-repudiation". 
+
+Roughly speaking, "deniability" is the opposite of "non-repudiation", i.e., the
+property that it is impossible to prove to a third party that a message was sent
```

Comment:
The meaning is clear here, but "third party" is a bit odd in a group-messaging setting. Maybe (n+1)th party? 🤔

Reply:
I didn't see a much better way to phrase this, so I kept third party...